docs/guides/core-concepts/cypress-app.mdx

```diff
@@ -172,6 +172,36 @@ The title of each run is taken from the
 for that change, and clicking on it will take you to the corresponding run page
 in Cypress Cloud.
 
+### Debug
+
+<DocsImage
+  src="/img/guides/core-concepts/cypress-app/debug-page.png"
+  alt="Debug"
+></DocsImage>
+
+The Debug page allows you to easily debug failed CI test runs from within the
+Cypress app. From this page, you can review, rerun, and debug failed CI test
+runs that are recorded to Cypress Cloud (see [note above](#Overview)) – all
+without leaving Cypress.
+
+This feature eliminates the tedious process of switching between Cypress, your
+local editor, and CI output, in order to fix failed tests. It does this by
+enabling you to use the [Test Runner](#The-Test-Runner) to run only the tests
+that failed in your last recorded test run, as well as review screenshots,
+videos, and logs from your tests.
+
+The Debug page shows the latest completed [test run](/guides/cloud/runs) that
+matches your current locally checked out git commit. If no run is found for your
+current commit, then Cypress crawls your git tree to find the most recent run in
```

Comment:
While not inaccurate, `Cypress crawls your git tree` might sound a little more invasive than we'd like? Could just describe the end result here:

```suggestion
current commit, then Cypress shows the most recent run in
```